src/Rammbock/core.py

```diff
@@ -1028,3 +1029,44 @@ def get_server_unread_messages_count(self, server_name=None):
         """
         server = self._servers.get(server_name)
         return server.get_messages_count_in_buffer()
+
+    def close_client(self, name=None):
+        """Closes the client connection based on the `client_name`.
+
+        If no name provided it will close the current active connection.
+        You have to explicitly Switch Client after close when sending or
```

Comment:
Other keywords in the docs should be in backticks: `Switch Client`. The generated api docs will then contain working links to that kw,